src/App/Home/index.js

```diff
@@ -30,27 +30,34 @@ const Home = () => {
-                                </div>
-                                <i className="material-icons">arrow_forward</i>
-                            </Link>
-                        </div>
-                    ))}
-                    <Link to={routes.find(route => route.title === "Get started").path} className="cards cards-primary cards-wide m-3">
-                        <div className="cards-content flex-row align-items-center m-0 w-100">
-                            <i className="material-icons material-icons-outlined mr-3 v-flip">wb_incandescent</i>
-                            <span className="h4 mb-0">Get started by setting up the Design Guide!</span>
-                            <i className="material-icons material-icons-outlined ml-auto">arrow_forward</i>
-                        </div>
-                    </Link>
+                    {routes.map((route, i) => {
+                        if (!route.wideCard) {
+                            return (
+                                <React.Fragment key={route.title}>
+                                    <div key={route.title} className="col-12 col-sm-6 col-lg-3 d-flex">
+                                        <Link to={route.path} className="cards cards-primary">
+                                            <div className="cards-icon">
+                                                <i className="material-icons-outlined" >{route.icon.name}</i>
+                                            </div>
+                                            <div className="cards-content">
+                                                <span className="h4">{route.title === "Utilities" ? "Utility" : route.title }</span>
+                                                <span>{route.entryCardText}</span>
+                                            </div>
+                                            <i className="material-icons">arrow_forward</i>
+                                        </Link>
+                                    </div>
+                                    {i === routes.length - 1 &&
+                                        <Link to={routes.find(route => route.wideCard === true).path} className="cards cards-primary cards-wide m-3">
+                                            <div className="cards-content flex-row align-items-center m-0 w-100">
+                                                <i className="material-icons material-icons-outlined mr-3 v-flip">wb_incandescent</i>
+                                                <span className="h4 mb-0">Get started by setting up the Design Guide!</span>
+                                                <i className="material-icons material-icons-outlined ml-auto">arrow_forward</i>
+                                            </div>
+                                        </Link>}
+                                </React.Fragment>
+                            );
+                        }
+                    })}
```

Comment:
Ah, better! But it's still somewhat hard coded with the texts etc
The following suggestion is not at all perfect, but I was thinking something along the lines of this:
```suggestion
                    {routes.map((route, i) => (
                        <React.Fragment key={route.title}>
                            <div key={route.title} className={route.wideCard ? "col-12 d-flex" : "col-sm-6 col-lg-3 "}>
                                <Link to={route.path} className={`cards cards-primary ${route.wideCard && "cards-wide m-3"}`}>
                                    {route.wideCard
                                        ?
                                        <div className="cards-content flex-row align-items-center m-0 w-100">
                                            <i className="material-icons material-icons-outlined mr-3 v-flip">{route.icon.name}</i>
                                            <span className="h4 mb-0">{route.entryCardText}</span>
                                            <i className="material-icons material-icons-outlined ml-auto">arrow_forward</i>
                                        </div>
                                        :
                                        <div>
                                            <div className="cards-icon">
                                                <i className="material-icons-outlined" >{route.icon.name}</i>
                                            </div>
                                            <div className="cards-content">
                                                <span className="h4">{route.title === "Utilities" ? "Utility" : route.title }</span>
                                                <span>{route.entryCardText}</span>
                                            </div>
                                            <i className="material-icons">arrow_forward</i>
                                        </div>
                                    }
                                </Link>
                            </div>
                        </React.Fragment>
                    ))}
```

You would need to rearrange the way routes are imported, and also add the "entryCardText" property to the "get started" route, but then it will be a lot more dynamic and it will pave way for more "wide" cards in the future should it be needed.

Reply:
Not entirely sure why the "Utilities" route is being hard coded into "Utility". Perhaps it can be removed? Would be even cleaner

Reply:
Your solution worked! I just added an `.order-5` to move the element to the bottom, instead of rearranging the imports. Rearranging the imports would affect other areas, like the sidebar.